docs/attribution.mdx

```diff
@@ -19,134 +19,250 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
-- Hesse data from [Hessischen Verwaltung für Bodenmanagement und Geoinformation](https://gds.hessen.de/INTERSHOP/web/WFS/HLBG-Geodaten-Site/de_DE/-/EUR/ViewDownloadcenter-Start?path=Liegenschaftskataster/Hauskoordinaten%20ohne%20Postalische%20Angaben%20(txt)). Available under [DL-DE->Zero-2.0](https://www.govdata.de/dl-de/zero-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- North Rhine-Westphalia data from [Geobasis NRW](https://www.opengeodata.nrw.de/produkte/geobasis/lk/akt/gebref_txt/). Available under [DL-DE->Zero-2.0](https://www.govdata.de/dl-de/zero-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Rhineland-Palatinate data from [GeoBasis-DE/LVermGeo RP](https://lvermgeo.rlp.de/geodaten-geoshop/open-data). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Saarland data from [GeoBasis-DE/LVermGeo SL](https://geoportal.saarland.de/mapbender/php/mod_iso19139ToHtml.php?url=https%3A%2F%2Fgeoportal.saarland.de%2Fmapbender%2Fphp%2Fmod_dataISOMetadata.php%3FoutputFormat%3Diso19139%26id%3Dcfc63fce-d914-4486-8f4f-c3c0973ddb64). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Saxony data from [Staatsbetrieb Geobasisinformation und Vermessung Sachsen](https://www.geodaten.sachsen.de/downloadbereich-hauskoordinaten-4172.html). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Saxony-Anhalt data from [GeoBasis-DE/LVermGeo ST](https://www.lvermgeo.sachsen-anhalt.de/de/gdp-open-data.html). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Schleswig-Holstein data from [©GeoBasis-DE/LVermGeo SH](https://geodaten.schleswig-holstein.de/gaialight-sh/_apps/dladownload/lizenz.html). Available under [CC BY 4.0](https://geodaten.schleswig-holstein.de/gaialight-sh/_apps/dladownload/lizenz.html). Distributed by OpenAddresses. Accessed: 2025-02-16.
-- Thuringia data from [Freistaat Thüringen](https://geoportal.thueringen.de/gdi-th/download-offene-geodaten). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-02-16.
+- Baden-Württemberg data from [LGL](https://www.lgl-bw.de/Produkte/Liegenschaftskataster/Hauskoordinaten/). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- Berlin data from [Geoportal Berlin](https://daten.berlin.de/tags/amtlichen-hauskoordinaten). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2025-03-09.
```

Comment:
This source url is no longer available, maybe we point to an archived version? 

https://web.archive.org/web/20230122025439/https://daten.berlin.de/tags/amtlichen-hauskoordinaten

or

https://daten.berlin.de/datensaetze?q=hauskoordinaten&sort=score+desc%2C+metadata_modified+desc